cm_generate_makefile.bash

```diff
@@ -367,8 +367,8 @@ display_help_text() {
       echo "--disable-perftests: Do not build Kokkos Kernels performance tests"
       echo "--enable-perftests: build Kokkos Kernels performance tests (default)"
       echo "--deprecated-code             Enable deprecated code (disabled by default)"
-      echo "--enable-perfsuite: build Kokkos Kernels performance tests with
-RAJAPerf Suite"
+      echo "--enable-perfsuite: build Kokkos Kernels performance tests with RAJAPerf Suite"
```

Comment:
Actually this should really be removed as it is no longer support we are moving to google benchmark.
When the google benchmark work is closer to finalized we can enable options in script for it.

Reply:
Good to know, deleted the help message. I held off on deleting the actual parse handler in case scripts are still using it
? Or did you mean for me to delete that too?